README.md

```diff
@@ -1,22 +1,23 @@
 # Teuthology API
 
-A REST API to execute [teuthology commands](https://docs.ceph.com/projects/teuthology/en/latest/commands/list.html). 
+A REST API to execute [teuthology commands](https://docs.ceph.com/projects/teuthology/en/latest/commands/list.html).
 
 ## Setup
 
-#### Option 1: (teuthology docker setup)
+### Option 1: (teuthology docker setup)
```

Comment:
please add the instructions to start the server outside of docker container, it would be very helpful!

Reply:
I just added some - #25 needs to be fixed before we can actually use secrets, though.

Reply:
Oh, I see! Yep I'm able to start the server without `.env` file.